### DISCUSSION TITLE
3.2.0 RC2

### DISCUSSION THREAD

**Discussion by sfmskywalker** [elsa-workflows/elsa-studio#245] [Announcements]
## What's Changed
* Update icon + port provider for DownloadHttpFile activity by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/226
* Allow the user to select only the versions with 'usable as activity' … by @MariusVuscanNx in https://github.com/elsa-workflows/elsa-studio/pull/230
* Lazy Loading of Subgraphs for Workflow Instance Viewer by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/231
* Optimize rendering of ActivityWrapper. by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/233
* Implement deep linking into workflow instance subgraphs by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/236
* Adjust calculate-activity-size by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/238
* Refactor workflow execution log selection logic by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/240
* Update DesignerPathChangedArgs and method usages by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/242

**Full Changelog**: https://github.com/elsa-workflows/elsa-studio/compare/3.2.0-rc1...3.2.0-rc2

<hr /><em>This discussion was created from the release <a href='https://github.com/elsa-workflows/elsa-studio/releases/tag/3.2.0-rc2'>3.2.0 RC2</a>.</em>

**Comment by glime-ai** [comment]
The changes from 3.2.0-rc1 to 3.2.0-rc2 in the Elsa Studio repository include several pull requests that have been merged. Here's a summary of what's changed:

1. **Update icon + port provider for DownloadHttpFile activity**: This pull request, by @sfmskywalker, likely includes updates to the visual representation and possibly the functionality of the DownloadHttpFile activity within the workflow designer.

2. **Allow the user to select only the versions with 'usable as activity'**: By @MariusVuscanNx, this change seems to allow users to filter and select specific versions of workflows or activities that are marked as usable within the workflow designer.

3. **Lazy Loading of Subgraphs for Workflow Instance Viewer**: This pull request, by @sfmskywalker, introduces lazy loading for subgraphs in the workflow instance viewer, which could improve performance and user experience when dealing with complex workflows.

4. **Optimize rendering of ActivityWrapper**: Also by @sfmskywalker, this change aims to optimize the rendering process of the ActivityWrapper component, potentially improving the designer's performance.

5. **Implement deep linking into workflow instance subgraphs**: This pull request, by @sfmskywalker, adds the ability to deep link directly into specific parts of a workflow instance's subgraphs, enhancing navigation and usability.

6. **Adjust calculate-activity-size**: By @sfmskywalker, this change likely involves adjustments to how the size of activities is calculated within the workflow designer, which could affect layout and visual aspects.

7. **Refactor workflow execution log selection logic**: This pull request, by @sfmskywalker, refactors the logic for selecting execution logs, which could improve the way logs are accessed and displayed within the studio.

8. **Update DesignerPathChangedArgs and method usages**: Also by @sfmskywalker, this change updates the arguments for the DesignerPathChanged event and its method usages, which could affect how path changes within the designer are handled and propagated.

For more details on each of these changes, you would typically visit the provided links to the pull requests on GitHub. However, since I cannot access external content, I cannot provide direct links or further details beyond the summary above.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
